ogr/services/github/flag.py

```diff
@@ -45,6 +45,8 @@ def _from_raw_commit_flag(self):
         self.comment = self._raw_commit_flag.description
         self.url = self._raw_commit_flag.target_url
         self.uid = self._raw_commit_flag.id
+        self.created = self._raw_commit_flag.created_at
+        self.edited = self._raw_commit_flag.updated_at
```

Comment:
What about implementing this in the property as well so it's loaded on-demand when called?

It can look like this:
- abstract -> property with `raise NotImplementedError`
- github -> property with `return self._raw_commit_flag.created_at`
- pagure -> same as github, property using `_raw_commit_flag`
- gitlab -> Nothing, or raise `NotImplementedError` with the linked issue